requirements.txt

```diff
@@ -13,3 +13,6 @@
 #
 # You can read more about pip-compile command here:
 # https://github.com/jazzband/pip-tools#example-usage-for-pip-compile
+
+-e git+git://github.com/inveniosoftware/invenio-accounts.git@rest-api#egg=invenio-accounts
```

Comment:
Please keep pushing for merging/releasing this and do not forget to remove this branches.